__tests__/app.test.js

```diff
@@ -125,3 +125,76 @@ describe("app", () => {
     });
   });
 });
+
+describe("app", () => {
+  describe("/api/articles/:article_id/comments", () => {
+    it("200: GET - Responds with an array for an article that has any comments", () => {
+      return request(app)
+        .get("/api/articles/6/comments")
+        .expect(200)
+        .then((response) => {
+          ({ comments } = response.body);
+          expect(comments).toBeInstanceOf(Array);
+        });
+    });
+
+    it("200: GET - Responds with the correct array length for an article that has more than 1 comment", () => {
+      return request(app)
+        .get("/api/articles/9/comments")
+        .expect(200)
+        .then((response) => {
+          ({ comments } = response.body);
+          expect(comments).toHaveLength(2);
+        });
+    });
+
+    it("200: GET - Responds with the array of object/s that have the following properties: comment_id, body, article_id, author, votes: created_at", () => {
+      return request(app)
+        .get("/api/articles/6/comments")
+        .expect(200)
+        .then((response) => {
+          ({ comments } = response.body);
+          comments.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+            });
+          });
+        });
+    });
+
+    it("200: GET - Responds with the array of comments in descending order if article has more than 1 comment", () => {
+      return request(app)
+        .get("/api/articles/9/comments")
+        .expect(200)
+        .then((response) => {
+          ({ comments } = response.body);
+          expect(comments).toBeSortedBy("created_at", { descending: true });
```

Comment:
Great test for the sort, this could all be done within a single test but I like that you've broke it out to follow TDD and make debugging easier